rtmbot/core.py

```diff
@@ -61,6 +68,16 @@ def connect(self):
     def _start(self):
         self.connect()
         self.load_plugins()
+        for plugin in self.bot_plugins:
+            try:
+                self._dbg("Registering jobs for {}".format(plugin.name))
```

Comment:
Yeah, that's a good question. I did that for backwards compatibility purposes, but given that this is already a large breaking change, I'm going to throw in that change as well. Given how simple is is to do the different INFO, DEBUG, ERROR logging, we should probably support it.